maro/simulator/scenarios/finance/common/common.py

```diff
@@ -0,0 +1,252 @@
+from abc import ABC, abstractmethod
+from enum import Enum
+from typing import Callable
+
+
+class OrderMode(Enum):
+    market_order = "market_order"
+    limit_order = "limit_order"
+    stop_order = "stop_order"
+    stop_limit_order = "stop_limit_order"
+    cancel_order = "cancel_order"
+
+
+class OrderDirection(Enum):
+    buy = "buy"
+    sell = "sell"
+
+
+class ActionType(Enum):
+    order = "order"
+    cancel_order = "cancel_order"
+
+
+class ActionState(Enum):
+    pending = "pending"
+    success = "success"
+    failed = "failed"
+    expired = "expired"
+    canceled = "canceled"
+
+
+class TradeResult:
+    """Result or a trade order"""
+
+    def __init__(
+            self, direction: OrderDirection, trade_number: int, price_per_item: float,
+            commission: float, tax: float
+    ):
+        self.trade_number = int(trade_number)
```

Comment:
If the ``trade_number`` is limited to be an integer, make sure ``trade_number = int(trade_number)`` here